src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC } from 'react';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: FC<Props> = ({ people }) => {
+  const newPeople = people.map((person: Person) => {
+    const newPerson: Person = { ...person };
+
+    const mother = people.find((el) => el.name === person.motherName);
```

Comment:
You don't need to create a separate const `mother` inside map function, you can directly use `person.motherName` in the following code, same for `father` const